kstreams-live-update/docker-compose-postgres.yaml

```diff
@@ -30,9 +30,9 @@ services:
     environment:
      - PG_MODE=primary
      - PG_PRIMARY_PORT=5432
-     - PG_PRIMARY_USER=postgres
```

Comment:
Will PORT above need updating, too?

Reply:
Seems to work fine, but probably because the port is the default port.

Reply:
I don't see any reference to the port in the docs